### PR TITLE
Animation for movement and combat

### PR DESCRIPTION
**Pull request by johnw08**
Added animation for movement and combat. Attack during movement will stop the movement.

Combat key:
J: light attack
K: heavy attack
U: left punch
I: right punch
B: buff

### CODE REVIEW

**Comment by jialuogao** [comment]
Please update the PR name so that it is meaningful instead of "Yw"

**Comment by johnw08** [comment]
> Please update the PR name so that it is meaningful instead of "Yw"

Done it.

**Review by DKevi — DISMISSED** [review]
LGTM

File: Assets/Character Script/BasicCombat.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
+using System.Collections;
```
Comment (jialuogao) [inline Assets/Character Script/BasicCombat.cs:1]: 如果不需要就删掉吧

File: Assets/Character Script/BasicCombat.cs.meta [hunk 2]
```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
```
Comment (jialuogao) [inline Assets/Character Script/BasicCombat.cs.meta:1]: 如果不需要就删掉吧

File: Assets/Character Script/BasicMoving.cs [hunk 3]
```diff
@@ -74,21 +72,46 @@ private void ApplyRotation()
...
+         
     }
 
     public void Move(InputAction.CallbackContext context)
     {
         _input = context.ReadValue<Vector2>();
-        Debug.Log(_input);
+        //Debug.Log(_input);
         _direction = new Vector3(_input.x, 0.0f, _input.y);
+        _direction_temp = _direction;
```
Comment (jialuogao) [inline Assets/Character Script/BasicMoving.cs]: 有用吗？

File: Assets/Character Script/BasicMoving.cs [hunk 4]
```diff
@@ -74,21 +72,46 @@ private void ApplyRotation()
...
+        //anim.SetTrigger("run");
+        //if (anim.GetCurrentAnimatorStateInfo(0))
```
Comment (jialuogao) [inline Assets/Character Script/BasicMoving.cs]: 没用的删掉吧

**Review by jialuogao — CHANGES_REQUESTED** [review]
minor

**Review by jialuogao — APPROVED** [review]

**Pull request merged**